### PR TITLE
Fix namespace util

### PR DESCRIPTION
**Pull request by barmac**
Builds upon https://github.com/camunda/camunda-modeler/pull/1227.

New namespace util will parse only definitions to look for namespace.

### CODE REVIEW

**Comment by barmac** [comment]
@nikku Let's hold it as we might want to add regex escape here.

**Comment by barmac** [comment]
@nikku ready

**Comment by barmac** [comment]
@nikku ping

**Comment by nikku** [comment]
@barmac I've applied the changes as discussed today (cf. https://github.com/camunda/camunda-modeler/pull/1279/commits/3b03aaea7a5e195ae889e73fce793080016cffc1). Please have a look :eyeglasses:.

File: client/src/app/tabs/bpmn/__tests__/BpmnEditorSpec.js [hunk 1]
```diff
@@ -544,81 +544,63 @@ describe('<BpmnEditor>', function() {
...
-        try {
-          expect(onContentUpdated).to.be.calledOnce;
-          expect(onContentUpdated).to.be.calledWith(activitiConvertedXML);
-
-          done();
-        } catch (error) {
-          done(error);
-        }
+        expect(onContentUpdated).to.be.calledOnce;
+        expect(onContentUpdated).to.be.calledWith(activitiConvertedXML);
```
Comment (barmac) [inline client/src/app/tabs/bpmn/__tests__/BpmnEditorSpec.js]: In this situation, it is not certain that the expect statements will be run. The callback may run after tests and then any errors are swallowed (false negative).

**Pull request merged**